File: client/src/app/_models/admin/customerReview.ts
import { ICustomerReviewItem } from "./customerReviewItem";

export interface ICustomerReview
{
     id: number;
     customerId: number;
     customerName: string;
     city: string;
     currentStatus: string;
     remarks: string;
     customerReviewItems: ICustomerReviewItem[];
}

File: api/Entities/Admin/Client/CustomerReview.cs
namespace api.Entities.Admin.Client
{
    public class CustomerReview: BaseEntity
    {
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public string City { get; set; }
        public string CurrentStatus {get; set;} = "Active";
        public string Remarks { get; set; }
        public ICollection<CustomerReviewItem> CustomerReviewItems { get; set; }
    }
}